content/shmem_sync_all.tex

```diff
@@ -0,0 +1,44 @@
+\color{ForestGreen}
+\apisummary{
+    Registers the arrival of a \ac{PE} at a barrier and suspends \ac{PE}
+    execution until all other \acp{PE} arrive at the barrier.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_sync_all(void);
+\end{Csynopsis}
+
+\begin{apiarguments}
+
+    \apiargument{None.}{}{}
+
+\end{apiarguments}
+
+\apidescription{
+    The \FUNC{shmem\_sync\_all} routine registers the arrival of a \ac{PE} at a
+    barrier. Barriers are a fast mechanism for synchronizing all \acp{PE} at
```

Comment:
I think "barrier" is the accepted terminology for this type of operation, so I'm ok with leaving it as-is.  The second sentence is defining barrier, which seems non-redundant.